modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/EditObjectField.tsx

```diff
@@ -107,10 +109,29 @@ export default function EditObjectField({
 		onSubmit,
 	});
 
+	const [objectDefinition, setObjectDefinition] = useState<
+		Partial<ObjectDefinition> | ObjectDefinition
+	>({enableLocalization: false});
+
+	useEffect(() => {
+		const makeFetch = async () => {
+			if (objectDefinitionExternalReferenceCode) {
+				const objectDefinitionResponse = await API.getObjectDefinitionByExternalReferenceCode(
+					objectDefinitionExternalReferenceCode
+				);
+
+				setObjectDefinition(objectDefinitionResponse);
+			}
+		};
+
+		makeFetch();
+	}, [objectDefinitionExternalReferenceCode]);
+
```

Comment:
Hey @igor-franca, you can move all this logic and the `useEffect` below too, into `EditObjectFieldContent.tsx`. This way we will declare useEffect only once and not in `EditObjectField.tsx` and `RightSidebarObjectFieldDetails.tsx`. 

Furthermore, you can combine the logic into just one `useEffect` :wink: